app/Http/Controllers/admin/RoleController.php

```diff
@@ -77,6 +77,12 @@ public function assignPermission(Request $request, Role $role)
         return back()->with('message', 'Permission added to role');
     }
 
+    public function syncPermissions(Request $request, Role $role){
+
+        $role->syncPermissions($request->permissions);
```

Comment:
What if a permission is selected that does not exist?

Reply:
THen it won't be added